### DISCUSSION TITLE
Can this plug in show Ring Contact Door/Window Sensors in HomeKit?

### DISCUSSION THREAD

**Discussion by larrybradshaw-thinkdifferent** [dgreif/ring#1200] [Q&A] [answered]
I see my Ring Cameras and security system base but not individual contact sensors.

**Comment by smockle** [comment, marked as answer]
Yes, this plugin supports contact sensors. If you’re using the latest version of iOS, sensors won’t appear (by default) on the main page of Apple’s Home app, but you should see them if you tap into a specific room. On the room page, sensors don’t appear as tiles, like other accessories; instead, they appear in a narrow horizontal band at the top.

**Comment by larrybradshaw-thinkdifferent** [comment]
Thank you! Found them.




Larry Bradshaw

Think Different Designs

------------------------------

Phone 321-421-9086  Mobile 515-991-0546

Email ***@***.***

Web thinkdifferentdesigns.com
